crates/rpc/src/method/simulate_transactions.rs

```diff
@@ -1741,8 +1741,13 @@ pub(crate) mod tests {
         );
     }
 
+    #[rstest::rstest]
+    #[case::v07(RpcVersion::V06)]
```

Comment:
OK, I see one change, but there should have been two - line 1745 above also has `#[case::v07(RpcVersion::V06)]`...

Reply:
Indeed. Again, that's me vs computer: did not save the file _before_ committing ;)

Should be fixed now.